GoogleForCreators/web-stories-wp
# Editor: Add storybook story for icons

## Summary

This brings the icons library in the editor on par with the dashboards', making it easier to import them and test them in storybook.

## Relevant Technical Choices



## To-do



## User-facing changes

None

## Testing Instructions

1. `npm run storybook`
1. Check out icons
1 In WordPress, open the editor
1. Verify nothing is broken
